PerformanceTests/JetStream2/JetStreamDriver.js

```diff
@@ -597,11 +636,71 @@ class Benchmark {
             Realm.dispose(magicFrame);
     }
 
+    async loadBlob(file) {
+        if (!window.blobDataCache)
+            window.blobDataCache = { };
+
+        let cache = window.blobDataCache;
+        let blobData = cache[file];
+        if (blobData) {
+            blobData[3]++;
+            return blobData;
+        }
```

Comment:
How about chaining a promise of blob in the cache instead?
Then we do not need to request the same file multiple times.
```

let promise = cache[file];
if (promise)
    return await promise;
...
promise = new Promise((resolve) => {
    let tries = 3;
    ...

    When fetch resource is resolved, call resolve(blob);
});
cache[file] = promise;
```

Reply:
I can try it.